src/OrbitGl/BatchRenderGroup.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitGl/BatchRenderGroup.h"
+
+namespace orbit_gl {
+StencilConfig& StencilConfig::ClipAt(const StencilConfig& parent) {
+  if (!parent.enabled) return *this;
+  if (!enabled) {
```

Comment:
I'm not sure if I get what means that a child is disabled, probably this came from a previous PR, so I will ping you offline.

Reply:
Ok - to clarify, it's not the child that is disabled, it's just that it does not have any StencilConfig set, so basically there is no clipping to the child's body happening. This is why we only need to care about the parent config.